src/Microsoft.FeatureManagement/IFilterParametersBinder.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+using Microsoft.Extensions.Configuration;
+
+namespace Microsoft.FeatureManagement
+{
+    /// <summary>
+    /// An interface used by the feature management system to pre-bind feature filter parameters to a settings type.
+    /// <see cref="IFeatureFilter"/>s can implement this interface to take advantage of caching of settings by the feature management system.
+    /// </summary>
+    public interface IFilterParametersBinder
```

Comment:
We can make this public in the future. For the release, I would hide it as internal for now so we can change our minds later.

Reply:
@zhenlan

Thoughts?

Reply:
@rossgrambo 

Discussed with @zhenlan. We agreed to have it public.